local/src/test/scala/com/salesforce/op/local/OpWorkflowModelLocalTest.scala

```diff
@@ -123,7 +123,7 @@ class OpWorkflowModelLocalTest extends FlatSpec with TestSparkContext with TempD
     }
     log.info(s"Scored ${expectedScores.length * numOfRuns} records in ${elapsed}ms")
     log.info(s"Average time per record: ${elapsed.toDouble / (expectedScores.length * numOfRuns)}ms")
-    elapsed should be <= 25000L
+    elapsed should be <= 50000L
```

Comment:
This change is not related. Let's omit this from this PR.

Reply:
@tovbinm  this is a flapper test that sometimes fails with 25000L as the threshold. That's why I increased it to 50000L

Reply:
Let's not double it. Let's increase it a little, e.g. 35000